Jenkinsfile

```diff
@@ -40,11 +42,25 @@ pipeline {
         HTTP_DIGEST_ACL=credentials("portaljs.${env.CF_SPACE}.http.digest.acl")
       }
       steps {
+        sh 'if [[ "${LOGSTASH_CONNECTION}" != "" ]]; then echo "services:\\n  - {CF_LOG_DRAINER_SERVICE}" >> manifest.yml; fi'
         sh 'sed -i "s|env:|env:\\n  CTF_CPA_ACCESS_TOKEN: ${CTF_CPA_ACCESS_TOKEN}|" manifest.yml'
         sh 'sed -i "s|env:|env:\\n  HTTP_DIGEST_ACL: ${HTTP_DIGEST_ACL}|" manifest.yml'
         sh 'cf blue-green-deploy ${CF_APP_NAME} -f manifest.yml --delete-old-apps'
       }
     }
+    stage('Register with ELK') {
+      when {
+        expression { return env.LOGSTASH_CONNECTION }
+      }
+      environment {
+        CF_APP_NAME="portaljs${env.CF_SPACE == 'production' ? '' : '-' + env.CF_SPACE}"
+      }
+      steps {
+        sh 'APP_GUID="$(cf app ${CF_APP_NAME} --guid | head -1)"'
+        sh 'RESULT=(ssh ${LOGSTASH_CONNECTION} /etc/logstash/register_appname_guid.sh ${APP_NAME} ${APP_GUID})'
```

Comment:
1. Why is the result of this command being assigned to `RESULT`?
2. This is very specific to our current Jenkins config. The path to that shell script should not be in this Pipeline script.

Reply:
1. This was copied from the API deployment script. I've confirmed with Patrick it's not needed.
2. Yes, as mentioned in Jira I also wasn't sure about this. I've now moved the entire command to a new variable(ELK_REGISTRATION_COMMAND), this removes the need for LOGSTASH_CONNECTION. So that's been removed.